### ISSUE TITLE
Tag v0.21.0 doesn't build

### ISSUE THREAD

**Issue by p0nce** [BindBC/bindbc-sdl#27]
```
C:\Users\guill\AppData\Local\dub\packages\bindbc-sdl-0.21.0\bindbc-sdl\source\bindbc\sdl\bind\sdlgamecontroller.d(219,9): Error: undefined identifier `SDL_GameControllerButton`
C:\Users\guill\AppData\Local\dub\packages\bindbc-sdl-0.21.0\bindbc-sdl\source\bindbc\sdl\bind\sdlgamecontroller.d(220,9): Error: undefined identifier `SDL_GameControllerButton`
C:\Users\guill\AppData\Local\dub\packages\bindbc-sdl-0.21.0\bindbc-sdl\source\bindbc\sdl\bind\sdlgamecontroller.d(221,9): Error: undefined identifier `SDL_GameControllerButton`
C:\Users\guill\AppData\Local\dub\packages\bindbc-sdl-0.21.0\bindbc-sdl\source\bindbc\sdl\bind\sdlgamecontroller.d(222,9): Error: undefined identifier `SDL_GameControllerButton`
```

Workaround: using v0.20.0

**Comment by mdparker** [comment]
My test app builds fine. Are you building static or dynamic? Which SDL version are you specifying?

**Comment by mdparker** [comment]
Okay. I see. It fails with anything less than SDL_2012.

**Comment by mdparker** [comment]
Fixed in 0.20.1. Thanks!
